manifests/config.pp

```diff
@@ -6,11 +6,17 @@
 class icingadb::config {
   assert_private()
 
+  $stdlib_version  = $icingadb::globals::stdlib_version
   $conf_dir        = $icingadb::globals::conf_dir
   $user            = $icingadb::globals::user
   $group           = $icingadb::globals::group
   $redis_tls_files = $icingadb::redis_tls_files
   $db_tls_files    = $icingadb::db_tls_files
+  $config_content  = if versioncmp($stdlib_version, '9.0.0') < 0 {
+    to_yaml($icingadb::config)
```

Comment:
are there still users for this old version of stdlib? metadata.json lists 9.0.0 als oldest supported version.

Reply:
Yes, I have a customer this week who is not on the bleeding edge.

Reply:
That's the reason for the other three Icinga modules to also support older version.

Reply:
If that version is supported, `metadata.json` should reflect that

Reply:
It does?!
https://github.com/voxpupuli/puppet-icingadb/blob/54fa5822604f95978ac978e7bd0ce50711aa2526/metadata.json#L12

Reply:
Sorry, I went @bastelfreak's comment

Reply:
@ekohl You're welcome.